src/renderer/src/misc/formatters.h

```diff
@@ -0,0 +1,23 @@
+#include <fmt/format.h>
+
+#include <glm/detail/qualifier.hpp>
+
+template <glm::length_t L, typename T, glm::qualifier Q>
+struct fmt::formatter<glm::vec<L, T, Q>> {
+	static constexpr auto parse(format_parse_context& ctx)
+			-> decltype(ctx.begin()) {
+		return ctx.end();
+	}
```

Comment:
I already seen same function somewhere, maybe you should make it template and move to consts?